src/translate.js

```diff
@@ -101,14 +94,17 @@ export default singleton => {
   }
 
   function insertInterpolations(translation, options) {
-    const regularInterpolations = pickBy(
-      options,
-      (val, key) => !has(defaultOptions, key) && !React.isValidElement(val)
-    )
+    let regularInterpolations = {}
+
+    for (const [key, value] of Object.entries(options)) {
+      if (!has(defaultOptions, key) && !React.isValidElement(value)) {
```

Comment:
The `has` implementation is a bit overkill for what we actually need here. What we want is to pull out the value of `markdown` from options.

Let's simplify this to:
```suggestion
      if (key !== 'markdown' && !React.isValidElement(value)) {
```